cabal-install/Distribution/Client/CmdConfigure.hs

```diff
@@ -83,7 +86,13 @@ configureAction (configFlags, configExFlags, installFlags, haddockFlags)
     baseCtx <- establishProjectBaseContext verbosity cliConfig
 
     -- Write out the @cabal.project.local@ so it gets picked up by the
-    -- planning phase.
+    -- planning phase. If old config exists, then print the contents
+    -- before overwriting
+    exists <- doesFileExist "cabal.project.local"
+    when exists $ do
+        notice verbosity "'cabal.project.local' file already exists. Now overwriting it."
+        contents <- readFile "cabal.project.local"
```

Comment:
We should really be using `copyFile` here. @hvr mentioned possible interference with `LOCALE` for `readFile`/`writeFile`.  http://hackage.haskell.org/package/directory-1.3.1.1/docs/System-Directory.html#v:copyFile